tests/test_target_database.py

```diff
@@ -1,46 +1,39 @@
 """Tests for `mbed_targets`."""
 
-import unittest
-import requests_mock
+import pytest
 
 # Unit under test
 import mbed_targets._internal.target_database as target_database
 
 
-class TestGetTargetData(unittest.TestCase):
+class TestGetTargetData:
```

Comment:
Please stick to using unittest.TestCase, we have already had the discussion around this area and the conclusion was the test case assertions were more descriptive and the standard already used in Mbed OS.

Reply:
Personally, I completely disagree that the `self.assert` style is clearer than using plain `assert` statements. Most of the python community also agrees these days. However, I don't want to rock the boat too much (I probably have already :trollface: ) - I will revert to the `unittest` style.

Reply:
I've moved back to `unittest` style in the latest commit. Let me know what you think.